e2e/app/page/workspace-data-page.ts

```diff
@@ -144,4 +147,37 @@ export default class WorkspaceDataPage extends WorkspaceBase {
     return analysisPage.createNotebook(notebookName, lang);
   }
 
+  // rename cohort-review thru the Ellipsis menu located inside the cohort-review card
+
+  async renameCohortReview(cohortName: string, newCohortReviewName: string): Promise<void> {
```

Comment:
Instead create a new function, you can modify `renameResource` function in `workspace-base` for `Cohort Review`.

Reply:
Done!